### ISSUE TITLE
Product Search Page

### ISSUE THREAD

**Issue by RyanYoak** [DataByne/uraPointOfSale#67]
Make a page that allows the user to search all products by name.

Dependent on #5

**Comment by kyletpugh** [comment]
What about by price? Can the user search for products only between a range of prices? Are there other inherent attributes that are searchable? Can the search results be sorted in some way?

**Comment by RyanYoak** [comment]
I can add that or do you want it to be added as a separate issue to modify this one, if we added everything in this search issue it might get too complex

**Comment by kyletpugh** [comment]
You kinda expanded more than the user story, but that is fine as this is the logical conclusion of the usage. But I was merely pointing out that there are many other traits associated directly in the products table that could probably be used to search in addition to the product name. You can modify this issue to include that, or create other issues. Anything outside of that where there is an additional table (like custom attributes and categories) should be added as a separate issue. You have already done that for categories, up to you if you want to create new issues for the others or not. They can always be added later...
